src/sonic-yang-models/yang-models/sonic-mux-cable.yang

```diff
@@ -0,0 +1,76 @@
+module sonic-mux-cable {
+    namespace "http://github.com/Azure/sonic-mux-cable";
+    prefix mux_cable;
+    yang-version 1.1;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC DualToR MUX CABLE confifuration data";
+
+    revision 2022-08-19 {
+        description
+            "Initial revision";
+    }
+
+    container sonic-mux-cable {
+        
+        container MUX_CABLE {
+            
+            list MUX_CABLE_LIST {
+                
+                key "ifname";
+
+                leaf ifname {
```

Comment:
Does mux cable have any dependency on interface?
For example, maybe we need to shutdown interface before configuring mux cable?

Reply:
MUX_CABLE table is per interface. 

Why do we need to do that?

Reply:
You can use leafref, please refer to https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-lldp.yang#L111-L117